aviator-terraform-taint.yml

```diff
@@ -0,0 +1,17 @@
+spruce:
+- base: ci/terraform-taint/meta-terraform-taint.yml
+  prune:
+    - meta
+  merge:
+    - with_in: ci/terraform-taint/
+      regexp: ".*yml"
+    - with_in: ci/terraform-taint/jobs/
+      regexp: ".*yml"
+    - with_in: ci/shared/
+      regexp: "meta.yml"
+  to: aviator_pipeline_terraform-taint.yml
+fly:
+  name: terraform-taint
+  target: concourse
```

Comment:
is this the correct target?

Reply:
It matches the pattern used in every other pipeline in the repo.